### PR TITLE
Document Pokémon item effects, other minor changes.

### PR DESCRIPTION
**Pull request by luckytyphlosion**
- Add optional RTLFLAGS variable in Makefile
- Use MEMORY in linkerscript, also pad elf file
- Add replace.sh

### CODE REVIEW

**Comment by huderlem** [comment]
emerald has a matching ItemEffects function, so it should be ported from there.

**Comment by PikalaxALT** [comment]
This can probably be closed.

File: ld_script.txt [hunk 1]
```diff
@@ -3,18 +3,21 @@ ENTRY(Start)
 gNumMusicPlayers = 4;
 gMaxLines = 60;
 
+MEMORY
```
Comment (PikalaxALT) [inline ld_script.txt]: This could probably be its own PR

File: ld_script.txt [hunk 2]
```diff
@@ -1226,4 +1224,18 @@ SECTIONS {
     {
         *(*);
     }
+    
+    .fillrom :
```
Comment (PikalaxALT) [inline ld_script.txt]: No need for this if you use objcopy with --gap-fill 0xFF
